nnbuilder/src/main/kotlin/org/hse/nnbuilder/utils/HashUtil.java

```diff
@@ -0,0 +1,39 @@
+package org.hse.nnbuilder.utils;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.StandardCopyOption;
+import org.apache.commons.codec.digest.DigestUtils;
+import org.hse.nnbuilder.exception.HashError;
+import org.jetbrains.annotations.NotNull;
+
+/**
+ * Helps to get a hash of file and hash of a String using sha1Hex
+ */
+public class HashUtil {
+    public static String getFileHash(@NotNull Path filePath) throws HashError {
```

Comment:
What's the advantage of declaring the throw here?
It should be sufficient to throw RuntimeError (or a derivative) without declaring `throws`.